src/probnum/filtsmooth/gaussfiltsmooth/kalman.py

```diff
@@ -295,10 +309,17 @@ def smooth_step(
                 start,
                 stop,
                 intermediate_step=intermediate_step,
+                diffusion=diffusion,
             )
 
     def _smooth_step_classic(
-        self, unsmoothed_rv, smoothed_rv, start, stop, intermediate_step=None
+        self,
+        unsmoothed_rv,
+        smoothed_rv,
+        start,
+        stop,
+        intermediate_step=None,
```

Comment:
Any preferences on the parameter order in `Kalman`: `filter_step`, `smooth_step` etc?

Reply:
not really. I think the "internal ones" (i.e. intermediate step, diffusion, etc.) should go behind the standard ones (start, stop, etc) but other than that, just pop new ones to the end of the list? I dont think this makes a big difference